CONTRIBUTING.md

```diff
@@ -9,13 +9,17 @@ In general, all code must follow the [WordPress Coding Standards and best practi
 - **WordPress**: As of Performance Lab v3.3.0, released July 15, 2024, the plugin's minimum WordPress version requirement is 6.5.
 - **PHP**: Always match the latest WordPress version. The minimum required version right now is 7.2.
 
-
 ## Guidelines
 
 - As with all WordPress projects, we want to ensure a welcoming environment for everyone. With that in mind, all contributors are expected to follow our [Code of Conduct](https://make.wordpress.org/handbook/community-code-of-conduct/).
-
 - All WordPress projects are [licensed under the GPLv2+](/LICENSE), and all contributions to Gutenberg will be released under the GPLv2+ license. You maintain copyright over any contribution you make, and by submitting a pull request, you are agreeing to release that contribution under the GPLv2+ license.
 
+## Use of AI Tools
+
+You are free to use artificial intelligence (AI) tooling to contribute, but we ask that you disclose what tooling you are using and to what extent a pull request has been authored by AI. Are you using AI just as a code reviewer? Or, for the other extreme, are you using AI to write everything (e.g. "vibe coding")? It is your responsibility to review and take responsibility for what AI generates.
```

Comment:
maybe we should add a question about AI use as part of the [Pull Request template ](https://github.com/WordPress/performance/blob/trunk/.github/PULL_REQUEST_TEMPLATE.md)?

Reply:
oh i missed that going top to bottom ;)